osu.Game/Screens/Select/Leaderboards/Leaderboard.cs

```diff
@@ -164,5 +276,85 @@ protected override void Update()
                 }
             }
         }
+
+        private class RetryButton : BeatSyncedContainer

```

Comment:
please remove this bouncing for now. the scale on hover is fine but the bouncing is ugly.

you can probably make this container an `: OsuClickableContainer` itself as a result